cloud_AWS/terraform/module/s3.tf

```diff
@@ -34,7 +34,7 @@ resource "aws_flow_log" "vpc_logs" {
     (var.s3_flowlogs_path == "" ? "${aws_s3_bucket.vpc_logs[count.index].arn}/" : "${aws_s3_bucket.vpc_logs[count.index].arn}/${var.s3_flowlogs_path}/") :
   (var.s3_flowlogs_path == "" ? "${aws_s3_bucket.vpc_logs[0].arn}/${var.vpc_id_list[count.index]}/" : "${aws_s3_bucket.vpc_logs[0].arn}/${var.s3_flowlogs_path}/${var.vpc_id_list[count.index]}/"))
   log_destination_type     = "s3"
-  log_format               = "$${version} $${account-id} $${interface-id} $${srcaddr} $${dstaddr} $${srcport} $${dstport} $${protocol} $${packets} $${bytes} $${start} $${end} $${action} $${log-status}"
+  log_format               = "$${version} $${account-id} $${interface-id} $${srcaddr} $${dstaddr} $${srcport} $${dstport} $${protocol} $${packets} $${bytes} $${start} $${end} $${action} $${log-status} $${vpc-id} $${subnet-id} $${instance-id} $${tcp-flags} $${type} $${pkt-srcaddr} $${pkt-dstaddr} $${region} $${az-id} $${sublocation-type} $${sublocation-id} $${pkt-src-aws-service} $${pkt-dst-aws-service} $${flow-direction} $${traffic-path}"
```

Comment:
Ted, does Transfluo require specific log format? If not, I suggest to not specify the format. That way, if AWS adds new fields, we will get them without having to rev. this module and reconfigure all existing flow exports.

Reply:
Agree.  The Terraform currently specifics only v2 fields.  An appropriate method of configuring AWS, to include all fields would be optimal.

The call out here is that the current configuration executed by Terraform will only writes v2 flow logs.  We need more fields than simply v2 fields.

Reply:
Got it. If Transfluo can handle any format that Amazon can produce, we can simply remove the `log_format` directive. It is optional and I assume that the default is to produce the most up-to-date format that Amazon offers. It would be of course good to verify it.

@TedatKentik, could you, please, test this change? Suggested steps:
- remove the line containing the `log_format` directive in your local copy
- create a TF resource file using the local copy (starting for example for this [main.tf](https://github.com/kentik/config-snippets-cloud/blob/master/cloud_AWS/terraform/module/examples/single-vpc/main.tf))
- create a new Amazon cloud export using the above created resource file
- run some traffic in the VPC for which you've created the new cloud export
- check that we are getting all data that we should be getting.